notebooks/guides/getting-started/v2/tutorial_pt6_v2.ipynb

```diff
@@ -0,0 +1,590 @@
+   "source": [
+    "# Compute dot product similarity and display results\n",
+    "n = 5\n",
+    "scores = np.dot(query_emb, np.transpose(doc_emb))[0]\n",
+    "max_idx = np.argsort(-scores)[:n]\n",
+    "\n",
+    "retrieved_documents = [faqs_long[item] for item in max_idx]\n",
+    "\n",
+    "for rank, idx in enumerate(max_idx):\n",
+    "    print(f\"Rank: {rank+1}\")\n",
+    "    print(f\"Score: {scores[idx]}\")\n",
+    "    print(f\"Document: {retrieved_documents[rank]}\\n\")\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Reranking can boost the results from semantic or lexical search further. The Rerank endpoint takes a list of search results and reranks them according to the most relevant documents to a query. This requires just a single line of code to implement.\n",
+    "\n",
+    "We call the endpoint using `co.rerank()` and pass the following arguments:\n",
+    "\n",
+    "- `query`: The user query\n",
+    "- `documents`: The list of documents we get from the semantic search results\n",
+    "- `top_n`: The top reranked documents to select\n",
+    "- `model`: We choose Rerank English 3\n",
+    "\n",
+    "Looking at the results, we see that the given a query about getting to know the team, the document that talks about joining Slack channels is now ranked higher (1st) compared to earlier (3rd).\n",
+    "\n",
+    "Here we select `top_n` to be 2, which will be the documents we will pass next for response generation."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 33,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Rank: 1\n",
+      "Score: 0.0020507434\n",
+      "Document: {'text': 'Joining Slack Channels: You will receive an invite via email. Be sure to join relevant channels to stay informed and engaged.'}\n",
+      "\n",
+      "Rank: 2\n",
+      "Score: 0.0014158706\n",
+      "Document: {'text': 'Team-Building Activities: We foster team spirit with monthly outings and weekly game nights. Feel free to suggest new activity ideas anytime!'}\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Rerank the documents\n",
+    "results = co.rerank(query=queries_for_search,\n",
+    "                    documents=retrieved_documents,\n",
+    "                    top_n=2,\n",
+    "                    model='rerank-english-v3.0')\n",
+    "\n",
+    "# Display the reranking results\n",
+    "for idx, result in enumerate(results.results):\n",
+    "    print(f\"Rank: {idx+1}\") \n",
+    "    print(f\"Score: {result.relevance_score}\")\n",
+    "    print(f\"Document: {retrieved_documents[result.index]}\\n\")\n",
+    "    \n",
+    "reranked_documents = [retrieved_documents[result.index] for result in results.results]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Finally we reach the step that we saw in the earlier `Basic RAG` section.\n",
```

Comment:
I don't think 'Basic RAG' should be in code-tense.

Reply:
done